walk/config_test.go

```diff
@@ -24,3 +30,53 @@ func TestCheckPathMatchPattern(t *testing.T) {
 		}
 	}
 }
+
+func TestConfigurerFlags(t *testing.T) {
+	dir, err := os.MkdirTemp(os.Getenv("TEST_TEMPDIR"), "config_test")
```

Comment:
What does this directory do? I don't see it used below.

(Simplification: you can use `t.TempDir()` to create a directory. It automatically handles errors and cleanup.)

Reply:
I simply copy+pasted the test from `config/config_test.go` and then deleted the parts unrelated to `ValidBuildFileNames`. I've never really looked into these tests and would rather not when just moving code.